app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,225 @@
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_weight="4"
+            android:orientation="vertical"
+            android:background="@color/third_layout_color"
+            tools:ignore="MissingConstraints">
+
+
+            <LinearLayout
+                android:id="@+id/fourteen"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                tools:ignore="MissingConstraints">
+
+                <CheckBox
+                    android:id="@+id/fifteen"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="5"
+                    android:text="@string/notify_me"
+                    android:checked="false" />
+
+                <Switch
+                    android:id="@+id/sixteen"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="5"
+                    android:text="@string/auto_backup"
+                    android:checked="false" />
+
+            </LinearLayout>
+
+            <View
+                android:id="@+id/seventeen"
+                android:layout_width="match_parent"
+                android:layout_height="0.5dp"
+                android:background="@color/line_color" />
+
+            <RatingBar
+                android:id="@+id/eighteen"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:progressTint="@color/rating_bag_progress_color"
+                android:numStars="5"
+                android:rating="4"
+                android:scaleX="0.6"
+                android:scaleY="0.6"
+                android:translationX="-40dp"
+                android:stepSize="1" />
+
+            <View
+                android:id="@+id/nineteen"
+                android:layout_width="match_parent"
+                android:layout_height="0.5dp"
+                android:background="@color/line_color" />
+
+            <RadioGroup
+                android:id="@+id/twenty"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content">
+
+                <RadioButton
+                    android:id="@+id/twentyone"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:checked="false"
+                    android:text="YES" />
+
+                <RadioButton
+                    android:id="@+id/twentytwo"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:checked="false"
+                    android:text="NO" />
```

Comment:
Please consider avoiding the hardcoded strings. Add them in strings.xml. Use CTRL + Enter to speed up the process.